src/utils/application.ts

```diff
@@ -0,0 +1,75 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Form, RetrievedApplication, State } from '@/types/application.types';
+
+function formatApplicationState(state: State) {
+  if (!state) return '';
+
+  const s = state.split('/')?.pop() || '';
+  return s.charAt(0).toUpperCase() + s.slice(1);
+}
+
+function isApplicationComplete(application: RetrievedApplication) {
+  if (!application.forms) return false;
+
+  return application.forms
+    .map((form) => form?.fields)
+    .flat()
+    .every((field) => field?.value?.split(',')?.length > 1);
```

Comment:
same here you don't need to do field?

Also based on your type definition field will also always have value, so you don;t need to put question mark for value?